numpy/core/code_generators/generate_umath.py

```diff
@@ -1137,6 +1144,13 @@ def english_upper(s):
           TD(O),
           signature='(n?,k),(k,m?)->(n?,m?)',
           ),
+'isalpha' :
+    Ufunc(1, 1, False_,
+          docstrings.get('numpy.core.umath.isalpha'),
+          None,
+          [TypeDescription(U, EmptyFunctionTypeDescr, U, '?'),
+           TypeDescription(S, EmptyFunctionTypeDescr, S, '?')],
```

Comment:
Could just leave this empty, instead of the `EmptyFunctionTypeDescr`?  If not, fine, but maybe add a comment to `EmptyFunctionTypeDescr` to explain what it is used for.

Yes, this ensures it shows up in `.types` and it may actually be a reasonable way to ensure that in practice.  But TBH, it is half-private anyway, so maybe we can just ignore it for now.

Reply:
This wasn't possible before cause of a syntax error on empty arrays under MSVC, but I wrote some code so that it's okay.